Build failure on 32 bit arm

See https://github.com/seanyoung/cir/issues/3#issuecomment-1222624442 :

```
   Compiling evdev v0.12.0
error[E0308]: mismatched types
   --> /home/keshlamIR/.cargo/registry/src/github.com-1285ae84e5963aae/evdev-0.12.0/src/lib.rs:393:29
    |
393 |     let dur = Duration::new(tv.tv_sec.unsigned_abs(), tv.tv_usec as u32 * 1000);
    |                             ^^^^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`
    |
help: you can convert a `u32` to a `u64`
    |
393 |     let dur = Duration::new(tv.tv_sec.unsigned_abs().into(), tv.tv_usec as u32 * 1000);
    |                                                     +++++++
```

On 32 bit arm `time_t` is an `i32`, see https://docs.rs/libc/latest/src/libc/unix/linux_like/linux/gnu/b64/mod.rs.html#20

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -390,7 +390,7 @@ fn systime_to_timeval(time: &SystemTime) -> libc::timeval {
 }
 
 fn timeval_to_systime(tv: &libc::timeval) -> SystemTime {
-    let dur = Duration::new(tv.tv_sec.unsigned_abs(), tv.tv_usec as u32 * 1000);
+    let dur = Duration::new(tv.tv_sec as u64, tv.tv_usec as u32 * 1000);
     if tv.tv_sec >= 0 {
         SystemTime::UNIX_EPOCH + dur
     } else {

